pyccel/ast/datatypes.py

```diff
@@ -1,5 +1,9 @@
 # coding: utf-8
-
+# pylint: disable=missing-module-docstring
```

Comment:
Please do not disable missing docstrings outside of the tests/ folder.

This file has a clear enough purpose for a docstring to be written